### PR TITLE
Get back prepareResult for gpx approximation

### PR DESCRIPTION
**Pull request by RZR-UA**

### CODE REVIEW

File: native/src/routePlannerFrontEnd.cpp [hunk 1]
```diff
@@ -589,6 +589,7 @@ bool RoutePlannerFrontEnd::findGpxRouteSegment(SHARED_PTR<GpxRouteApproximation>
 		gctx->routeCalculations++;
 		RoutingContext* cp = new RoutingContext(gctx->ctx);
 		res = searchRouteInternalPrepare(cp, start->pnt, target->pnt, nullptr);
+		prepareResult(cp, res);
```
Comment (vshcherb) [inline native/src/routePlannerFrontEnd.cpp:592]: we need sync java & C++ so prepareResult should be inside searchRouteInternalPrepare

Comment (RZR-UA) [reply native/src/routePlannerFrontEnd.cpp:592]: this is just **hotfix** for current master. makePrecise/prepareResult should be fixed everywhere, in separate request(s)

Comment (vshcherb) [reply native/src/routePlannerFrontEnd.cpp:592]: Hotfix is in place. searchRouteInternalPrepare should run prepare inside function
- Based on function name
- Cause Java has it inside

**Comment by RZR-UA** [comment]
will by fixed by another request

**Pull request closed without merging**